SYCL/ESIMD/api/functional/value.hpp

```diff
@@ -112,6 +112,19 @@ template <typename DataT> struct value {
       return std::numeric_limits<DataT>::max();
     }
   }
+
+  static DataT ulp(DataT base_val, DataT direction) {
+    if constexpr (std::is_same_v<DataT, sycl::half>) {
+      return static_cast<sycl::half>(
+          (sycl::nextafter(base_val, direction) - base_val) * 8192);
```

Comment:
Would you please add a comment here. Why 8192 is used for sycl::half ?

Reply:
comment was added in this commit: a756ab394043ea46d847f0e2d30d7139c13eecb2